packages/ripple-address-codec/src/xrp-codec.ts

```diff
@@ -258,3 +251,9 @@ export function isValidClassicAddress(address: string): boolean {
   }
   return true
 }
+
+function checkByteLength(bytes: ByteArray, expectedLength: number): boolean {
```

Comment:
Had to add this checker because arrays and Uint8Array length's are different.  Looking back encodeChecked was typed to just `Buffer` but was expanded in the noble libs PR.  If keep it more strictly typed we can eliminate this checking logic.

Reply:
Is this function used in multiple files? (If so, can we pull it into a shared spot?)